### DISCUSSION TITLE
amamba/user-guide/wizard/create-app-git/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1747] [Announcements]
# amamba/user-guide/wizard/create-app-git/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/amamba/user-guide/wizard/create-app-git/

**Comment by liyuerich** [comment]
如果基于git仓库创建微服务应用输入项填写错误，怎么修改？

**Comment by liyuerich** [comment]
目标 镜像仓库有什么要求吗？比如公开仓库，私有仓库什么的？

**Comment by liyuerich** [comment]
能介绍一下 “概览” 页面吗？
